llvm/lib/Target/EraVM/MCTargetDesc/EraVMInstPrinter.cpp

```diff
@@ -100,6 +100,9 @@ void EraVMInstPrinter::printCCOperand(const MCInst *MI, unsigned OpNo,
   case EraVMCC::COND_LE:
     O << ".le";
     break;
+  case EraVMCC::COND_OF:
```

Comment:
You also need to ensure this is properly encoded, not just printed.

Reply:
@TerryGuo could we not introduce this flag, OF is an alias of LT. Let's use LT.

Reply:
Hi @asl Thanks for comments. @akiramenai Sure, I will update patch to use LT.

Reply:
OF is replaced with LT now.